tests/integration/ha/helpers.py

```diff
@@ -593,3 +595,65 @@ async def assert_restore_indices_and_compare_consistency(
     # We expect that new_count has a loss of documents and the numbers are different.
     # Check if we have data but not all of it.
     assert 0 < new_count < original_count
+
+
+async def assert_upgrade_to_local(
```

Comment:
In a next PR - Please take this out of the HA helpers / modules. We should have a separate package for upgrades.

Reply:
Btw, did it now